portfolio/src/main/webapp/style.css

```diff
@@ -27,3 +27,7 @@
 .center{
   text-align: center;
 }
+
+#secret-button{
+    display: none;
```

Comment:
Keep indentation consistent throughout the file (2 spaces)

Reply:
Got it